pkg/tenant/api/v1/api.go

```diff
@@ -283,8 +283,14 @@ type ProjectTopicPage struct {
 }
 
 type ProjectQueryRequest struct {
-	ProjectID string `json:"project_id"`
-	Query     string `json:"query"`
+	ProjectID  string            `json:"project_id"`
+	Query      string            `json:"query"`
+	Parameters []*QueryParameter `json:"parameters"`
```

Comment:
Added the parameters as a placeholder for right now, I would need to think more about how to deal with the parameter types.

Reply:
I think parameters are most useful for the SDK where users are trying to sanitize user input -- we will probably not use parameters from Beacon, so it's not really something Tenant has to worry about. You don't have to remove the parameters if you want to leave the stubs in.